card/client.go

```diff
@@ -23,18 +29,16 @@ func New(params *stripe.CardParams) (*stripe.Card, error) {
 // New creates a new card.
 func (c Client) New(params *stripe.CardParams) (*stripe.Card, error) {
 	if params == nil {
-		return nil, errors.New("params should not be nil")
+		return nil, fmt.Errorf("params should not be nil")
 	}
 
 	var path string
-	if params.Account != nil {
-		path = stripe.FormatURLPath("/v1/accounts/%s/external_accounts",
-			stripe.StringValue(params.Account))
-	} else if params.Customer != nil {
-		path = stripe.FormatURLPath("/v1/customers/%s/sources",
-			stripe.StringValue(params.Customer))
+	if params.Customer != nil {
```

Comment:
Re-ordered these conditional cases. I _think_ these should all have either `Customer` or `Account`, so this should be fine.

Reply:
This is technically a behavior change since we don't enforce that exclusively one is set... this is generated as part of codegen though and so likely can't be ordered right?

Reply:
This part is actually hardcoded. Updated to match existing code!